examples/async/webpack.config.js

```diff
@@ -4,8 +4,7 @@ var webpack = require('webpack');
 module.exports = {
   devtool: 'cheap-module-eval-source-map',
   entry: [
-    'webpack-dev-server/client?http://localhost:3000',
-    'webpack/hot/only-dev-server',
+    'webpack-hot-middleware/client?path=/__webpack_hmr&timeout=20000',
```

Comment:
There is no reason to add `?path=/__webpack_hmr&timeout=20000` as the defaults are good enough and adding it here just confuses beginners.